components/kyma-environment-broker/internal/process/deprovisioning/connectivity_unbind.go

```diff
@@ -0,0 +1,60 @@
+package deprovisioning
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/process"
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/storage"
+	"github.com/sirupsen/logrus"
+
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal"
+)
+
+type ConnectivityUnbindStep struct {
+	operationManager *process.DeprovisionOperationManager
+}
+
+func NewConnectivityUnbindStep(os storage.Operations) *ConnectivityUnbindStep {
+	return &ConnectivityUnbindStep{
+		operationManager: process.NewDeprovisionOperationManager(os),
+	}
+}
+
+var _ Step = (*ConnectivityUnbindStep)(nil)
+
+func (s *ConnectivityUnbindStep) Name() string {
+	return "CONNECTIVITY_Unbind"
```

Comment:
change the name to something like: Connectivity_Unbind (the same for other steps)

Reply:
Oh, I thought that this is the convention there :D `SOMETHING_Unbind` - with the uppercase component name. Isn't it right?